docs/nodes/nodes-library/nodes/HackerNews/README.md

```diff
@@ -0,0 +1,39 @@
+---
+permalink: /nodes/n8n-nodes-base.hackerNews
+---
+
+# Hacker News
+
+[Hacker News](https://news.ycombinator.com/) is a social news website focusing on computer science and entrepreneurship. It is run by investment fund and startup incubator, Y Combinator.
```

Comment:
'It is run by investment fund and startup incubator, Y Combinator.' can be removed.